### PR TITLE
Require artwork_id and artwork_version_id on line items in exchange

### PR DESCRIPTION
**Pull request by lilyfromseattle**
# [PURCHASE-1212] 

We recently had a case where manually-edited order mistakenly didn't have `artwork_version_id` set on its line items. This caused a gallery's "completed" tab to error out.

We should be requiring `artwork_id` and `artwork_version_id` be present on line items in exchange. This will help prevent similar errors in the future.

[PURCHASE-1212]: https://artsyproduct.atlassian.net/browse/PURCHASE-1212

### CODE REVIEW

File: hokusai/test.yml [hunk 1]
```diff
@@ -8,10 +8,11 @@ services:
     ports:
       - 8080:8080
     environment:
-      - RAILS_ENV=test
-      - DATABASE_HOST=exchange-db
-      - DATABASE_USER=postgres
-      - REDIS_URL=redis://exchange-redis
+    - RAILS_ENV=test
```
Comment (ashkan18) [inline hokusai/test.yml]: is this indent change intentional? it may be ok but im always scared of YML

**Comment by joeyAghion** [comment]
Reassigning from @ashkan18 😞 

@lilyfromseattle it looks like you have the context about next steps. Feel free to reassign once ready.

**Comment by sweir27** [comment]
@lilyfromseattle this looks okay but I know you and @sepans had been working through the various migration steps! I will leave it to you two to merge/monitor as it may require coordination.

**Comment by sepans** [comment]
Thanks @sweir27 yes just wanted to confirm that the `migration` and the validation under `line_item` looks correct to you. Otherwise, we did the staging/production migrations and everything was good.

**Comment by sepans** [comment]
Merging this and keeping an 👁 on migration.

**Pull request merged**